### PR TITLE
Update group.md

### PR DESCRIPTION
**Pull request by yyuank**

### CODE REVIEW

**Comment by yyuank** [comment]
@Lauragra could you please advise why one of the checks failed? I just copied everything from beta.

**Comment by Lauragra** [comment]
@yyuank , the assigned label resource topic doesn't exist in the v1.0 folder but we have a link. Are you bringing assignedLabels to v1.0?

**Comment by yyuank** [comment]
@Lauragra Yes, we are bringing assignedLabels to V1.

**Comment by Lauragra** [comment]
> @Lauragra Yes, we are bringing assignedLabels to V1.

Okay, then please add an assignlabels.md resource file to the v1.0 folder.

**Comment by yyuank** [comment]
@Lauragra How do I add assignedlabels resource file to v1.0 folder?  I know how to add a link. Should I just add a link? Apologies for the beginner questions...

**Comment by Lauragra** [comment]
> @Lauragra How do I add assignedlabels resource file to v1.0 folder? I know how to add a link. Should I just add a link? Apologies for the beginner questions...

Just copy the MD file from the beta folder and add it to the v1.0 folder. Make sure that the content is still accurate for v1.0. We'll also need a changelog entry to indicate that this property is available in v1.0.

File: api-reference/v1.0/resources/group.md [hunk 1]
```diff
@@ -106,13 +106,15 @@ This resource supports:
...
 |deletedDateTime|DateTimeOffset| For some Azure Active Directory objects (user, group, application), if the object is deleted, it is first logically deleted, and this property is updated with the date and time when the object was deleted. Otherwise this property is null. If the object is restored, this property is updated to null. |
 |description|String|An optional description for the group. <br><br>Returned by default.|
 |displayName|String|The display name for the group. This property is required when a group is created and cannot be cleared during updates. <br><br>Returned by default. Supports $filter and $orderby. |
+|expirationDateTime|DateTimeOffset| Timestamp of when the group is set to expire. The value cannot be modified and is automatically populated when the group is created. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`. <br><br>Returned by default. Read-only. |
```
Comment (dkershaw10) [inline api-reference/v1.0/resources/group.md:117]: So how do I extend it?  Can we point folks at that topic?
Are all groups set to expire, or can this be null?

Comment (yyuank) [reply api-reference/v1.0/resources/group.md:117]: @dkershaw10 expirationDateTime is extended by renewing the group, similar to renewedDateTime. We have a feature doc that talks about the group expiration feature in general (https://docs.microsoft.com/en-us/azure/active-directory/users-groups-roles/groups-lifecycle), but I'm not sure if we want to link feature doc to this API doc. Not all unified groups are set to expire, only the groups under expiration policy, so the value for some groups can be null.

Comment (yyuank) [reply api-reference/v1.0/resources/group.md:117]: @dkershaw10 Once a user renews a group, expiationDateTime is computed by adding group life cycle (number of days) defined in the life cycle policy to renewedDateTime.

**Comment by yyuank** [comment]
Changelog PR: https://github.com/microsoftgraph/microsoft-graph-docs/pull/8531

**Pull request closed without merging**